sematic/db/tests/fixtures.py

```diff
@@ -105,6 +105,7 @@ def make_run(**kwargs) -> Run:
     id = uuid.uuid4().hex
     run = Run(
         id=id,
+        original_run_id=None,
```

Comment:
Is this necessary? Isn't it going to be set to `None` by default?

Reply:
I like to be explicit. It informs the reader that we haven't missed anything, and it also clearly exposes all options / information.